components/Pagination/sub-components/PageButton.jsx

```diff
@@ -8,6 +8,7 @@ import {
 } from '../../shared/theme';
 
 const PageButton = styled.a`
+  font-weight: bold;
```

Comment:
if you are going to fix the suggest that i gave, could you put this property in the correct alphabetical order?

Reply:
it will need a mobile hover event rule. And i think this is not good to implement.